django_cloud_deploy/nox.py

```diff
@@ -51,8 +51,8 @@ def unit_test(session, python_version):
 def lint(session):
     """Run linters."""
     session.interpreter = 'python3.5'
-    session.install('flake8')
-    session.run('flake8')
+    session.install('yapf')
+    session.run('yapf', '--diff', '-r', 'cli', 'cloudlib', 'workflow')
```

Comment:
Should we also check skeleton/ and tests/?

Reply:
I think that we should do tests (added) but skeleton should probably follow whatever style Django uses.

Reply:
I see.